File: 코딩 기초 트레이닝/주사위 게임2-p181930/주사위 게임2.py
def solution(a, b, c):
    answer = 0

    data = set([a,b,c])
    data = list(data)

    if len(data) == 3:
        return a+b+c
    elif len(data) == 2:
        return (a+b+c) * (a*a + b*b + c*c)
    else:
        return (a+b+c) * (a*a + b*b + c*c) * (a*a*a + b*b*b + c*c*c)

File: 코딩 기초 트레이닝/주사위 게임2-p181930/주사위 게임2.js
function solution(a, b, c) {
    var answer = 0;
    const set = new Set([a,b,c]);
    console.log(set.size)

    if(set.size === 3) {
        return a + b + c;
    } else if(set.size === 2) {
        return (a + b + c) * ( a*a + b*b + c*c );
    } else {
        return (a + b + c) * ( a*a + b*b + c*c ) * (a*a*a + b*b*b + c*c*c);
    }


    return answer;
}
